docs/tutorials/polybeam_simulation.ipynb

```diff
@@ -0,0 +1,543 @@
+{
```

Comment:
<p>This profile doesn't seem particularly helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I'm going to get rid of one these polybeam notebooks and rewrite the other. I'll save the originals, maybe we can add them later if they are fixed up.